### DISCUSSION TITLE
How does the Toast work?

### DISCUSSION THREAD

**Discussion by Ri2parna** [zeit-ui/react#680] [Q&A] [answered]
I found the document a bit hard to understand and use the toast, do i have to add something like <Toaster /> at the top of the page? like we do with react-hot-toast?

**Comment by unix** [comment, marked as answer]
No extra work required, just use it. (In principle, this `Context` is actually loaded automatically in `GeistProvider`)

**Reply by unix** [reply]
PS: there may be some issues in version 2.2.3 that affect the `Toast` component (#679), I suggest you [upgrade to v2.2.4](https://github.com/geist-org/geist-ui/releases/tag/v2.2.4):
```bash
yarn add @geist-ui/core@latest
```
